cargo-dist/tests/snapshots/error_manifest.snap

```diff
@@ -6,11 +6,6 @@ stdout:
 {"diagnostic": {"message": "This workspace doesn't have anything for cargo-dist to Release!","severity": "error","causes": [],"labels": [],"related": []}}
 
 stderr:
-analyzing workspace:
-  cargo-dist (didn't match tag v1.0.0-FAKEVERSION)
```

Comment:
Maybe we still want to print this one?

Reply:
If we want to preserve it in normal runs then we probably need to do the "store it and have the caller choose to print it" thing (you can `writeln!` to a String, I'm sure there's an example somewhere in this codebase...)

Reply:
We don't actually pass the info this one line is keyed off of during the `check` runs, so we could just print it to stderr.

Reply:
There's several different conditions though, some of them static (publish=false, dist=false...)

Reply:
Hmm, right. Well, in the interest of getting this fix out, let's just leave it like this for now.